src/utils/bolt11.ts

```diff
@@ -22,3 +22,17 @@ export const getAmountAndExpiryFromInvoice = (invoice: string) => {
 
    return { amount, expiry };
 };
+
+/**
+ *
+ * @param invoice invoice to decode
+ * @returns
+ */
+export const decodeInvoice = (invoice: string) => {
+   const decoded = bolt11Decoder.decode(invoice);
+
+   const amountSat = Number(decoded.sections[2].value / 1000);
+   const expiryUnixSeconds = decoded.expiry + decoded.sections[4].value;
```

Comment:
what's going on here with this addition?

Reply:
This function is just easier to use and makes more sense. Then I can add other fields to decode backwards compatible later. I didn't want to go and replace all the uses of the above two functions in this PR, but eventually that would be nice.